pom.xml

```diff
@@ -125,18 +120,18 @@
             <optional>true</optional>
         </dependency>
 
-        <!-- Google Commons -->
+        <!-- Apache library for easily handling Collections -->
         <dependency>
-            <groupId>commons-collections</groupId>
-            <artifactId>commons-collections</artifactId>
-            <version>3.2.2</version>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.1</version>
         </dependency>
 
-        <!-- Curator is not compatible with Guava > 20.0 :( -->
+        <!-- Google's Guava library, makes life easier through common tooling -->
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>20.0</version>
```

Comment:
If I recall this used to break Curator?  Is this no longer the case?

Reply:
It probably does, but I'm going to work through the curator stuff next.